package.json

```diff
@@ -1,16 +1,20 @@
 {
     "name": "nsromania-setup",
-    "version": "0.1.0",
+    "version": "0.1.1",
     "scripts": {
         "dev": "next dev",
         "lint": "next lint",
         "build": "NEXT_PUBLIC_COMMIT_HASH=$(git rev-parse HEAD) next build",
-        "start": "next start"
+        "start": "next start",
+        "storybook": "storybook dev -p 6006",
+        "build-storybook": "storybook build"
     },
     "dependencies": {
         "@auth/prisma-adapter": "^2.8.0",
         "@emotion/react": "^11.14.0",
         "@emotion/styled": "^11.14.0",
+        "@fontsource/material-icons": "^5.1.1",
+        "@fontsource/roboto": "^5.1.1",
```

Comment:
Recommended by storybook to improve performance - automatically added when installed storybook